ShopifySharp/Entities/ProductListingVariant.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    public class ProductListingVariant
+    {
+        /// <summary>
+        /// The object's unique id.
+        /// </summary>
+        [JsonProperty("id")]
+        public long Id { get; set; }
```

Comment:
Once the `ShopifyObject` inheritance is added, this `Id` property can be removed entirely (it comes from the inheritance).

Reply:
Here and below inheritance `ShopifyObject` is implemented